asreview/webapp/src/ProjectComponents/SetupComponents/DataComponents/AddDataset.js

```diff
@@ -75,7 +75,7 @@ const AddDataset = (props) => {
               <code>.txt</code>) and tabular datasets (<code>.csv</code>,{" "}
               <code>.tab</code>, <code>.tsv</code>, <code>.xlsx</code>). The
               selected dataset should contain the title and abstract of each
-              record.{" "}
+              record and be free of any duplicates.{" "}
```

Comment:
What about this?

```suggestion
              record. It is highly recommended to upload a dataset without duplicate records.{" "}
```

Reply:
`...to add a dataset`
Be consistent with the wording on the page.

Reply:
Thanks

Reply:
Good! The addition of highly definitely makes it more urgent!